content/build/building/annotations.md

```diff
@@ -0,0 +1,190 @@
+```
+
+If you're using [Bake](../bake/_index.md), you can use the `annotations`
+attribute to specify annotations for a given target:
+
+```hcl
+  output = ["type=registry"]
+  annotations = ["foo=bar"]
+```
+
+For examples on how to add annotations to images built with GitHub Actions, see
+[Add image annotations with GitHub Actions](../ci/github-actions/annotations.md)
+
+You can also add annotations to an image created using `docker buildx
+imagetools create`. This command only supports adding annotations to an index
+or manifest descriptors, see
+[CLI reference](../../engine/reference/commandline/buildx_imagetools_create.md#annotations).
+
+## Inspect annotations
+
+To view annotations on an **image index**, use the `docker buildx imagetools
+inspect` command. This shows you any annotations for the index and descriptors
+(references to manifests) that the index contains. The following example shows
+an `org.opencontainers.image.documentation` annotation on a descriptor, and an
+`org.opencontainers.image.authors` annotation on the index.
+
+```console {hl_lines=["10-12","19-21"]}
+$ docker buildx imagetools inspect <IMAGE> --raw
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.index.v1+json",
+  "manifests": [
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "digest": "sha256:d20246ef744b1d05a1dd69d0b3fa907db007c07f79fe3e68c17223439be9fefb",
+      "size": 911,
+      "annotations": {
+        "org.opencontainers.image.documentation": "https://foo.example/docs",
+      },
+      "platform": {
+        "architecture": "amd64",
+        "os": "linux"
+      }
+    },
+  ],
+  "annotations": {
+    "org.opencontainers.image.authors": "dvdksn"
+  }
+}
+```
+
+To inspect annotations on a manifest, use the `docker buildx imagetools
+inspect` command and specify `<IMAGE>@<DIGEST>`, where `<DIGEST>` is the digest
+of the manifest:
+
+```console {hl_lines="22-25"}
+$ docker buildx imagetools inspect <IMAGE>@sha256:d20246ef744b1d05a1dd69d0b3fa907db007c07f79fe3e68c17223439be9fefb --raw
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "config": {
+    "mediaType": "application/vnd.oci.image.config.v1+json",
+    "digest": "sha256:4368b6959a78b412efa083c5506c4887e251f1484ccc9f0af5c406d8f76ece1d",
+    "size": 850
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
+      "digest": "sha256:2c03dbb20264f09924f9eab176da44e5421e74a78b09531d3c63448a7baa7c59",
+      "size": 3333033
+    },
+    {
+      "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
+      "digest": "sha256:4923ad480d60a548e9b334ca492fa547a3ce8879676685b6718b085de5aaf142",
+      "size": 61887305
+    }
+  ],
+  "annotations": {
+    "index,manifest:org.opencontainers.image.vendor": "foocorp",
+    "org.opencontainers.image.source": "https://git.example/foo.git",
+  }
+}
+```
+
+## Specify annotation level
```

Comment:
I see this section is almost the same as buildx one in https://github.com/docker/buildx/pull/2154/files#diff-c3a631f177fce518883a51cf79b3361da6ef527a09f55a8cac21991117a1425aR72

In buildx cli ref do you think we should cross link to docs website instead like https://github.com/docker/buildx/blob/0a0252d9b3a84ed7065eadd634a31e5c16bd1cbd/commands/build.go#L490?

Nevermind, actually I think that's fine.